exonum/src/blockchain/block.rs

```diff
@@ -22,36 +22,43 @@ pub const SCHEMA_MAJOR_VERSION: u16 = 0;
 encoding_struct!(
     /// Exonum block header data structure.
     ///
-    /// Block is essentially a list of transactions, which is
+    /// A block is essentially a list of transactions, which is
     /// a result of the consensus algorithm (thus authenticated by the supermajority of validators)
     /// and is applied atomically to the blockchain state.
     ///
-    /// Header only contains the amount of transactions and the transactions root hash as well as
+    /// The header only contains the amount of transactions and the transactions root hash as well as
     /// other information, but not the transactions themselves.
     struct Block {
-        /// Information schema version.
+        /// Returns the schema version.
```

Comment:
:stop_sign: :no_good_man: This is not a method, but rather a field; it does not return anything. It is common (both in Rust and in other languages) to document _methods_ with verb phrases ("Returns a value") and _fields_ with noun phrases ("A value").